rules/youtube-banner-show-upcoming-events/rule.md

```diff
@@ -0,0 +1,38 @@
+---
+type: rule
+archivedreason: 
+title: Does your home page banner show upcoming events?
+guid: 61d15bb1-aa8c-406c-97ea-4cd08bd22340
+uri: youtube-banner-show-upcoming-events
+created: 2023-08-11T03:56:22.0000000Z
+authors:
+- title: Seth Daily
```

Comment:
Author should be @jontygardner as it was his suggestion

Reply:
done